calico-cloud/about/index.mdx

```diff
@@ -35,5 +35,5 @@ The best fit for {{prodname}} is small teams who need to manage the full spectru
 - To connect your cluster to {{prodname}} in 15 minutes, [start a free trial](https://auth.calicocloud.io/u/signup/identifier?state=hKFo2SB3ekhybXN1TGdxTkZTUWIwQV9BSzNlaHBEUk0wMENJdKFur3VuaXZlcnNhbC1sb2dpbqN0aWTZIEE5b2NkREs1eWZKR0twc0ZWZmh2LWZCZEZxb2ZRNkJOo2NpZNkgc3NJQkNFdEdkZFpLNlVubDNOYWl2ZzhrY2RmcWd6dFE)
 - [Calico Cloud pricing](https://www.tigera.io/tigera-products/calico-cloud-pricing/)
 - [Connect a cluster to {{prodname}} documentation](../get-started/connect/connect-cluster.mdx)
-- [Image assurance documentation](../image-assurance/scan-image-registries.mdx)
+- [Image assurance documentation](../image-assurance)
```

Comment:
Does it matter whether the link has `.mdx` extension?

Reply:
It is fine to have .mdx extension.